.circleci/config.yml

```diff
@@ -69,6 +69,7 @@ jobs:
     # This is based on your 1.0 configuration file or project settings
     - run: pip install pipenv
     - run: pipenv install
+    - run: pipenv shell
```

Comment:
This should come before `pipenv install`. It still works as it is because `install` automatically creates the virtualenv, but it makes more sense to create and activate it ourselves (`pipenv shell`) if we already need this.